src/rust/scheduler/scheduler.rs

```diff
@@ -377,7 +396,7 @@ mod tests {
     }
 
     #[test]
-    fn scheduler_poll_twice() -> Result<()> {
+    fn poll_twice_with_one_long_task_completes_it() -> Result<()> {
```

Comment:
Could you add a Rust Doc comment for this test, like we have for other tests in this module?